docs/source/etl/variant-data.rst

```diff
@@ -206,15 +206,14 @@ file or glob pattern.
 The schema of the resulting DataFrame matches that of the VCF reader. The accompanying variant and sample information
 files must be located at ``{prefix}.bim`` and ``{prefix}.fam``.
 
-<<<<<<< HEAD
 +----------------------+---------+-----------------+-----------------------------------------------------------------------------------------------------+
 | Parameter            | Type    | Default         | Description                                                                                         |
 +======================+=========+=================+=====================================================================================================+
 | ``includeSampleIds`` | boolean | ``true``        | If true, each genotype includes the name of the sample ID it belongs to.                            |
 +----------------------+---------+-----------------+-----------------------------------------------------------------------------------------------------+
-| ``bimDelimiter``     | string  | `` `` (space)   | Whitespace delimiter in the ``{prefix}.bim`` file.                                                  |
+| ``bimDelimiter``     | string  | (space)         | Whitespace delimiter in the ``{prefix}.bim`` file.                                                  |
```

Comment:
I'm not sure how clear it is if we don't spell out what the string is, but I don't have a strong opinion either way.